Linux/template_datadog.json

```diff
@@ -1,5 +1,5 @@
 {
-    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
```

Comment:
Trailing `#` intended?

Reply:
Yes, [arm-ttk](https://github.com/Azure/arm-ttk) recommends a trailing `#`: https://github.com/Azure/arm-ttk/blob/32b40917e0ed6e9012b9da68b3eb5f358e8a990a/arm-ttk/testcases/deploymentTemplate/DeploymentTemplate-Schema-Is-Correct.test.ps1#L18-L23 
(not really sure why it recommends it though)